lib/dateTime.js

```diff
@@ -877,13 +877,18 @@ export function getDateTimeDescriptor() {
 			dateFormats = ['dddd, d MMMM yyyy', 'dd MMMM yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd MMMM', 'd MMM'];
 			break;
 		case 'fr-ca':
-		case 'fr-on':
 			dateFormats[1] = 'MMM d yyyy';
 			dateFormats[2] = 'yyyy-MM-dd';
 			dateFormats[4] = 'MMMM d';
 			dateFormats[5] = 'MMM d';
 			firstDayOfWeek = 0;
 			break;
+		case 'fr-on':
+			dateFormats[0] = 'dddd\' le \'d MMMM yyyy'; 
+			dateFormats[1] = 'MMM d yyyy';
+			dateFormats[2] = 'yyyy-MM-dd';
+			firstDayOfWeek = 0;
+			break;
```

Comment:
Not sure why I had to press a button to allow the CI to run... but it caught a linting error in here and I'm guessing the unit tests will fail due to the "le" change.

Reply:
Thanks, I just removed the trailing space. We'll see how this goes. 👍

Reply:
Oh I know why it needs me to approve your CI runs -- it's because this is a fork. If you're a member of the BrightspaceUI organization you should have Write access to the repo.

Reply:
As expected, some of the unit tests are failing. There are a couple that I wouldn't have expected (`4 juin` vs. `juin 4`) so probably something to have a look at there.

Reply:
I needed to change some tests in 'lms', too.

I have no experience with the BrightspaceUI code (I'm not a member!) so I'll take a look around for them.